.github/workflows/alteration-sequence.yml

```diff
@@ -0,0 +1,29 @@
+name: Alteration Sequence
+
+on:
+  push:
+    branches: [master]
+
+concurrency:
+  group: alteration-sequence-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  check-sequence:
```

Comment:
maybe we can put it in `main` but just another job?

Reply:
updated